### PR TITLE
revenue sharing POC

### PR DESCRIPTION
**Pull request by antomor**
PR to merge work between @mortelli and @antomor for the revenue sharing POC.

### CODE REVIEW

File: src/modals/Deploy.js [hunk 1]
```diff
@@ -108,10 +108,11 @@ function Deploy(props) {
     }
 
     async function handleDeploySmartWalletButtonClick() {
-        deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
+        // deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
         deploy.tokenGas = deploy.tokenGas === "" ? "0" : deploy.tokenGas;
 
         setLoading(true);
+        console.log(deploy.fees);
```
Comment (mortelli) [inline src/modals/Deploy.js]: we can take out this one

File: src/modals/Deploy.js [hunk 2]
```diff
@@ -108,10 +108,11 @@ function Deploy(props) {
     }
 
     async function handleDeploySmartWalletButtonClick() {
-        deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
+        // deploy.fees = deploy.fees === "" ? "0" : deploy.fees;
```
Comment (mortelli) [inline src/modals/Deploy.js]: sorry, why was this commented out? do we not need it because of state setting? if so, we can just remove the line

**Review by mortelli — COMMENTED** [review]
minor comments

File: src/modals/Deploy.js (hunk 2 above)
Comment (antomor) [reply src/modals/Deploy.js]: Solved, thank you for pointing it out!

**Review by mortelli — APPROVED** [review]

**Pull request merged**